src/rpc/rawtransaction.cpp

```diff
@@ -206,7 +206,16 @@ static UniValue gettxoutproof(const JSONRPCRequest& request)
 {
     if (request.fHelp || (request.params.size() != 1 && request.params.size() != 2))
         throw std::runtime_error(
-            "gettxoutproof [\"txid\",...] ( blockhash )\n"
+            RPCHelpMan{"gettxoutproof",
+                {
+                    RPCArg{"txids", RPCArg::Type::ARR,
```

Comment:
Can remove `RPCArg` throughout?

Reply:
@promag Done and thx! That makes it look less verbose and probably easier to read the documentation straight from the cpp file.